src/sessions/softphone-session-handler.ts

```diff
@@ -162,17 +162,27 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
 
     const communicationStateChanged = previousCallState?.state !== callState.state;
     let eventToEmit: boolean | SdkConversationEvents = 'updated';
+    const isOutbound = callState.direction === 'outbound'
 
     /* only check for emitting if the state changes */
     if (communicationStateChanged) {
       /* `pendingSession` – only process these if we have a persistent connection */
       if (this.isPendingState(callState)) {
+        // Not always accurate. If inbound auto answer, we don't know about it from convo evt
+
+        // headset actions will be tied to conversation updates rather than session events so we want to react regardless
+        if (isOutbound) {
+          this.sdk.headset.outgoingCall({ conversationId });
+        } else {
+          this.sdk.headset.setRinging({ conversationId, contactName: null }, !!this.lastEmittedSdkConversationEvent.current.length);
+        }
+
         /* only emit `pendingSession` if we already have an active session */
         if (session && session === this.activeSession) {
           const pendingSession: IPendingSession = {
             id: session.id,
             sessionId: session.id,
-            autoAnswer: callState.direction === 'outbound', // Not always accurate. If inbound auto answer, we don't know about it from convo evt
+            autoAnswer: isOutbound,
```

Comment:
Would this be accurate for ACD for example? I know we talked about it before but it feels strange that auto answering an incoming call through ACD would have the direction of outbound